### ISSUE TITLE
What website should i use when adding videos?

### ISSUE THREAD

**Issue by CubeFun** [cooolgamer/Nintendo-Anime-Channel-Revival#1]
I have tried every website like archive.org and wix.com but they are not working i tried to use the ssl patch too if u know any websites that i can use to upload the moflex files to let me know

**Comment by MicroshaftLLC** [comment]
Might be a bit silly, but have you tried a website where it doesn't force HTTPS, like 000webhost? I used that for my local testing of LetsShop, and it works so far.

**Comment by CubeFun** [comment]
Yes i did try to use 000webhost and it worked nicely but i tried to add episodes of kirby right back at ya but the free website only has 200 mb of storage and it can barely fit 1 episode and the ssl patch doesnt work for me
